packages/sync-actions/src/product-types-actions.js

```diff
@@ -21,12 +21,14 @@ export const baseActionsList = [
   { action: 'changeDescription', key: 'description' },
 ]
 
-export function actionsMapBase(diff, previous, next) {
+export function actionsMapBase(diff, previous, next, config) {
   return buildBaseAttributesActions({
     diff,
     actions: baseActionsList,
     oldObj: previous,
     newObj: next,
+    shouldOmitEmptyString: config.shouldOmitEmptyString,
+    // omitUnecessaryUpdates: config.shouldOmitEmptyString,
```

Comment:
I think you want to remove this comment ;)

Reply:
maybe I wasn't clear earlier, but I left that comment there as a suggestion to naming.
I would need help in finding a good name for this since `shouldOmitEmptyString` is just... bad 😄